packages/dashboard/src/pages/CreateMaskWallet/AddDeriveWallet/index.tsx

```diff
@@ -126,6 +127,7 @@ const AddDeriveWallet = memo(function AddDeriveWallet() {
                         `${walletName}${pathIndex}`,
                         mnemonic,
                         `${HD_PATH_WITHOUT_INDEX_ETHEREUM}/${pathIndex}`,
+                        false,
```

Comment:
I think only the core wallet manager is allowed to decide which wallet is imported or derived.